pkg/aws/db_elasticache.go

```diff
@@ -53,7 +53,7 @@ func listTaggedElasticacheDatabases(svc elasticache.ElastiCache, tagName string)
 			replicationGroupId = *cluster.ReplicationGroupId
 		}
 
-		_, ttl, isProtected, _, _ := utils.GetEssentialTags(tags.TagList, tagName)
+		_, ttl, isProtected, _, _ := common.GetEssentialTags(tags.TagList, tagName)
```

Comment:
For another review, but maybe worth to introduce a proper struct `EssentialTags` having `ttl`, `isProtected` and so on as fields, it would bring more clarity.